extensions/file-manager/src/utils.tsx

```diff
@@ -217,25 +209,3 @@ export function getDirectoryData(path: string): FileDataType[] {
   }
   return data;
 }
-
-export function Directory(props: { path: string }) {
```

Comment:
I moved this component to a separated file `extensions/file-manager/src/ui/Directory.tsx` since I wanted to reduce LOC in `extensions/file-manager/src/utils.tsx` file